packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -895,20 +907,12 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
 
   /* @conditional-compile-remove(DNS) */
   public async startNoiseSuppressionEffect(): Promise<void> {
-    const audioEffects: AudioEffectsStartConfig = {
-      noiseSuppression: new DeepNoiseSuppressionEffect()
-    };
-
-    await this.handlers.onStartNoiseSuppressionEffect(audioEffects);
+    await this.handlers.onStartNoiseSuppressionEffect();
```

Comment:
Just curious. Why don't we expose the function arguments for our adapter function?

Reply:
We are now providing the audioeffect through adapter option that is passed through the handlers, so we don't need to pass through adapters